gammagl/models/grade.py

```diff
@@ -0,0 +1,103 @@
+import tensorlayerx as tlx
+from gammagl.layers.conv import GCNConv
+import numpy as np
+
+
+# Multi-layer Graph Convolutional Networks
+class GCN(tlx.nn.Module):
```

Comment:
We already have GCN models in gammagl.models, so is this necessary?

Reply:
It can be implemented by GCNModel instead. However, in the original paper, the activation function used by the author on the Computers dataset is prelu and  GCNModel only has relu for activation function. So if we use GCNModel directly, the model will be slightly different from the original model.
In fact, the GRADE model has the same network structure as GRACE. The only difference is that GRADE uses a more complex  graph augmentation method . So my GRADE model is basically a copy of the GRACE model.

Reply:
OK, thanks for the work.